src/components/Scene.java

```diff
@@ -18,20 +19,15 @@
 
     @Override
     public void parse() {
-        // Check for scene name
         name = tokenizer.getNext();
-
-        // Check for texts
-        // Check for choices
-        // Check for timer
-        // Check for bgm
-        // Check for picture file; if picture exists, check for position
         while (true) {
             switch (tokenizer.checkNext()) {
                 case "text":
                     tokenizer.getAndCheckNext("text");
-                    texts.add(tokenizer.getNext());
-                    break;
+                    while (!Main.literals.contains(tokenizer.checkNext())) {
+                        texts.add(tokenizer.getNext());
+                        break;
+                    }
```

Comment:
Dunno if need another break here

Reply:
oops, lool, yeah i missed that heh, i should un approve it, but whatever LOL

Reply:
woops, it should be outside of the while loop, correct?